README.md

```diff
@@ -36,3 +53,7 @@ Give a ⭐️ if you like this project!
 ## 📝 License
 
 This project is [MIT](./MIT.md) licensed.
+
+## Design Credits
+
+- Design: [@Cindy Shin in Behance](https://www.behance.net/gallery/29845175/CC-Global-Summit-2015)
```

Comment:
- [ ] It would be better if you add acknowledgments to your README.
If you have used anyone else's work (designs, images, etc) that has a copyright that requires acknowledgment, you might want to add that to the acknowledgment section. You can also acknowledge any other developers or institutions who helped with the project.
Here 👇 is an example of a really good "Achnowledgments" section:
<img width="601" alt="image" src="https://user-images.githubusercontent.com/77190203/152332184-14fe5aa2-11c0-400d-842f-fb07c1f52d81.png">

Reply:
thanks Ill note it.